### PR TITLE
Supported iscsi and ceph images' convert

### PR DESCRIPTION
**Pull request by zhencliu**
1. Added cfg/remote_image_convert.cfg
2. The following testing scenarios are covered:
     remote -> local -> remote
     remote -> remote

ID: 1786234, 1786235

### CODE REVIEW

**Comment by zhencliu** [comment]
@lolyu

File: qemu/tests/cfg/remote_image_convert.cfg [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    variants:
+        - to_local:
+            # remote -> local(convert)
+            enable_iscsi_convert = no
+            enable_ceph_convert = no
+            image_raw_device_convert = no
+            storage_type_convert = filesystem
+        
+            # local(convert) -> remote(remote)
+            convert_target_remote = remote
```
Comment (lolyu) [inline qemu/tests/cfg/remote_image_convert.cfg]: the config name is a little misleading here, better to use `convert_target_convert = remote`?

**Comment by zhencliu** [comment]
@hellohellenmao 
--testcase=remote_image_convert

**Comment by hellohellenmao** [comment]
A warning while `git am`

`# git am 2018.patch
Applying: Supported iscsi and ceph images' convert
.git/rebase-apply/patch:85: trailing whitespace.
        
warning: 1 line adds whitespace errors.
`

**Comment by hellohellenmao** [comment]
Passed over ISCSI

`# python ConfigTest.py --testcase=remote_image_convert --machine=q35 --guestname=RHEL.8.1.1 --imageformat=luks --storage-backend-conf=/tmp/iscsi.cfg --clone=no --customsparams='qemu_force_use_drive_expression=no'`

` (01/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.q35: PASS (871.45 s)
 (02/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_qcow2.q35: PASS (903.26 s)
 (03/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_luks.q35: PASS (1143.67 s)
 (04/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_raw.q35: PASS (1057.77 s)
 (05/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_qcow2.q35: PASS (872.03 s)
 (06/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_luks.q35: PASS (1122.58 s)
 (07/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_raw.q35: PASS (1105.56 s)
 (08/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_qcow2.q35: PASS (1144.65 s)
 (09/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_luks.q35: PASS (1442.37 s)
 (10/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.q35: PASS (792.14 s)
 (11/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.none.q35: PASS (1249.01 s)
 (12/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.writeback.q35: PASS (1082.24 s)
 (13/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.unsafe.q35: PASS (1261.84 s)
 (14/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.directsync.q35: PASS (1515.45 s)
 (15/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.writethrough.q35: PASS (1800.22 s)
 (16/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_qcow2.q35: PASS (1114.13 s)
 (17/17) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_luks.q35: PASS (1447.97 s)
`

**Comment by zhencliu** [comment]
From Tingting:
1. cache_mode for remote_to_local_to_remote, one format is enough, e.g. raw
2. source_cache_mode = "none" when cache_mode is configured

**Comment by hellohellenmao** [comment]
Passed over Ceph.

`# python ConfigTest.py --testcase=remote_image_convert --machine=q35 --guestname=RHEL.8.1.0 --imageformat=raw --storage-backend-conf=/tmp/ceph.cfg --clone=no --customsparams='qemu_force_use_drive_expression=no'`

` (01/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.q35: PASS (1268.94 s)
 (02/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.none.q35: PASS (1252.72 s)
 (03/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.writeback.q35: PASS (1239.30 s)
 (04/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.unsafe.q35: PASS (1239.39 s)
 (05/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.directsync.q35: PASS (1245.75 s)
 (06/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.writethrough.q35: PASS (1292.22 s)
 (07/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_qcow2.q35: PASS (10929.49 s)
 (08/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_luks.q35: PASS (4090.44 s)
 (09/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_raw.q35: PASS (1208.60 s)
 (10/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_qcow2.q35: PASS (9863.37 s)
 (11/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_luks.q35: PASS (4525.60 s)
 (12/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_raw.q35: PASS (1880.82 s)
 (13/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_qcow2.q35: ERROR: Login timeout expired    (output: "exceeded 240 s timeout, last failure: Login timeout expired    (output: '')") (11582.62 s)
 (14/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_luks.q35: PASS (4532.68 s)
 (15/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.q35: PASS (1778.02 s)
 (16/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.none.q35: PASS (1762.32 s)
 (17/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.writeback.q35: PASS (1764.16 s)
 (18/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.unsafe.q35: PASS (1839.35 s)
 (19/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.writethrough.q35: PASS (1839.20s)
 (20/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.directsync.q35: PASS (1839s)
 (21/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_qcow2.q35: PASS (12061.45 s)
 (22/22) Host_RHEL.m8.u2.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_luks.q35: PASS (4242.41 s)
`

Note:
For the failed case, it's because a production bug under tracing.

**Comment by hellohellenmao** [comment]
> remote_image_convert.to_local.to_luks.to_remote.to_qcow2

For the failed case, it should be the known bug under tracing. Not about this patch, thanks.

**Comment by zhencliu** [comment]
without auth:
 (01/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw: PASS (685.66 s)
 (02/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.none: PASS (826.46 s)
 (03/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.writeback: PASS (688.34 s)
 (04/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.unsafe: PASS (662.04 s)
 (05/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.directsync: PASS (2027.43 s)
 (06/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_raw.cache_mode.writethrough: PASS (1947.01 s)
 (07/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_qcow2: PASS (731.66 s)
 (08/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_raw.to_remote.to_luks: PASS (844.81 s)
 (09/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_raw: PASS (687.86 s)
 (10/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_qcow2: PASS (691.99 s)
 (11/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_qcow2.to_remote.to_luks: PASS (754.54 s)
 (12/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_raw: PASS (827.62 s)
 (13/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_qcow2: PASS (823.89 s)
 (14/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_local.to_luks.to_remote.to_luks: PASS (999.19 s)
 (15/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw: PASS (633.19 s)
 (16/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.none: PASS (647.32 s)
 (17/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.writeback: PASS (649.04 s)
 (18/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.unsafe: PASS (683.16 s)
 (19/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.directsync: PASS (966.59 s)
 (20/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_raw.cache_mode.writethrough: PASS (958.79 s)
 (21/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_qcow2: PASS (764.16 s)
 (22/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.to_remote.to_luks: PASS (668.79 s)

**Comment by zhencliu** [comment]
@luckyh @xutian Can we merge the tc? I will also have to ping you guys in another 4 tcs to be merged, thank you.

File: qemu/tests/cfg/remote_image_convert.cfg [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            enable_ceph_convert = no
+            image_raw_device_convert = no
+            storage_type_convert = filesystem
+
+            # local(convert) -> remote(remote)
+            convert_target_remote = remote
+            image_name_remote = "images/image_convert_remote"
+            iscsi_direct:
+                lun_remote = 1
+        - to_remote:
```
Comment (luckyh) [inline qemu/tests/cfg/remote_image_convert.cfg]: What does this variants do? I see there are already a variant named `to_remote` defined at the above.

Comment (luckyh) [inline qemu/tests/cfg/remote_image_convert.cfg]: And basically, we should not have two anonymous variant that use the same name.

Comment (zhencliu) [reply qemu/tests/cfg/remote_image_convert.cfg]: The following testing scenarios are covered:
remote -> local -> remote
remote -> remote

So you can see two "to_remote"

Comment (zhencliu) [reply qemu/tests/cfg/remote_image_convert.cfg]: Removed the same variants, and performed the dryrun, which can generate the same tc list.

**Comment by zhencliu** [comment]
@luckyh @xutian 
Change done, only cfg file was changed, please help to merge the patch, thx.

**Comment by zhencliu** [comment]
(01/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_raw_convert: PASS (730.17 s)
 (02/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_raw_convert.cache_mode.none: PASS (803.28 s)
 (03/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_raw_convert.cache_mode.writeback: PASS (697.00 s)
 (04/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_raw_convert.cache_mode.unsafe: PASS (674.21 s)
 (05/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_raw_convert.cache_mode.directsync: PASS (2020.37 s)
 (06/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_raw_convert.cache_mode.writethrough: PASS (1950.82 s)
 (07/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_qcow2_convert: PASS (685.72 s)
 (08/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_raw_remote.fmt_luks_convert: PASS (830.75 s)
 (09/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_qcow2_remote.fmt_raw_convert: PASS (744.29 s)
 (10/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_qcow2_remote.fmt_qcow2_convert: PASS (686.37 s)
 (11/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_qcow2_remote.fmt_luks_convert: PASS (820.24 s)
 (12/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_luks_remote.fmt_raw_convert: PASS (841.94 s)
 (13/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_luks_remote.fmt_qcow2_convert: PASS (763.75 s)
 (14/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_local_to_remote.fmt_luks_remote.fmt_luks_convert: PASS (1023.03 s)
 (15/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_raw_convert: PASS (639.37 s)
 (16/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_raw_convert.cache_mode.none: PASS (654.34 s)
 (17/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_raw_convert.cache_mode.writeback: PASS (652.35 s)
 (18/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_raw_convert.cache_mode.unsafe: PASS (629.00 s)
 (19/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_raw_convert.cache_mode.directsync: PASS (955.08 s)
 (20/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_raw_convert.cache_mode.writethrough: PASS (957.76 s)
 (21/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_qcow2_convert: PASS (627.82 s)
 (22/22) Host_RHEL.m8.u2.product_av.iscsi_direct.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.remote_image_convert.remote_to_remote.fmt_luks_convert: PASS (703.14 s)
RESULTS    : PASS 22 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**